webpack/ForemanInventoryUpload/Components/PageHeader/components/PageDescription/PageDescription.js

```diff
@@ -55,14 +55,26 @@ export const PageDescription = () => {
         </Text>
       )}
       <Text ouiaId="text-minimal-data-collection">
-        {__('Learn more about ')}
-        <a
-          href={getDocsURL('Managing_Hosts', 'setting-minimal-data-collection')}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {__('Minimal data collection setting')}
-        </a>
+        <FormattedMessage
+          id="minimal-data-collection-message"
+          defaultMessage={__(
+            'Learn more about {minimalDataCollectionSetting}.'
+          )}
+          values={{
+            minimalDataCollectionSetting: (
+              <a
+                href={getDocsURL(
+                  'Managing_Hosts',
+                  'setting-minimal-data-collection'
+                )}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {__('Minimal data collection setting')}
```

Comment:
Now that we're here, I can nitpick the wording too :smile: 

I would say either "setting minimal data collection." or "the minimal data collection setting." That way you have a complete sentence:
```
Learn more about setting minimal data collection.
Learn more about the minimal data collection setting.
```

Reply:
+1